### PR TITLE
Add section to give an idea of when to use IDAM API

### PR DESCRIPTION
**Pull request by gellner**
- Should not be put into live until OpenStack is migrated to IDAM

### CODE REVIEW

**Comment by gellner** [comment]
Should we include text which says "if you can't do this or have problems then raise a Request ticket?

Is it possible/advisable to add a line which gives an estimate when IDAM permissions will be controllable via a Web UI (in the portal)?

**Comment by gellner** [comment]
When more platforms (esp in different regions/zones) are SSO/IDAM connected then there will probably be need for a separate page which says which products and which regions are IDAM-connected.

File: articles/other/other-ref-idam.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ UKCloud's Identity & Access Management API (IDAM) enables you to manage user acc
 
 Full documentation for the IDAM API is available on [SwaggerHub](https://app.swaggerhub.com/apis/ukcloud/idam-service).
 
+## Platforms using IDAM
```
Comment (suehighmoor) [inline articles/other/other-ref-idam.md]: We tend to reserve the word platform to refer to the UKCloud platform as a whole. Could we change this to:

UKCloud services using IDAM

File: articles/other/other-ref-idam.md [hunk 2]
```diff
@@ -20,6 +20,15 @@ UKCloud's Identity & Access Management API (IDAM) enables you to manage user acc
...
+
+IDAM is in use on the UKCloud for OpenStack plaforms, and for newer UKCloud for OpenShift deployments.
```
Comment (suehighmoor) [inline articles/other/other-ref-idam.md]: Suggest changing to:

IDAM is used for UKCloud for OpenStack and for newer UKCloud for OpenShift clusters.

Comment (boardfish) [reply articles/other/other-ref-idam.md]: Personally I'd prefer:

> IDAM is compatible with UKCloud for OpenStack and newer UKCloud for OpenShift clusters.

**Comment by suehighmoor** [comment]
@gellner, where are we with this article?

**Pull request closed without merging**